INSTALL.md

```diff
@@ -119,6 +123,10 @@ To build the JavaScript files for production, run the command:
 
     yarn build
 
+## Helm Installation (for deployment to Kubernetes)
+
+See [INSTALL-HELM.md](INSTALL_HELM.md) for guidance on this method.
```

Comment:
Would it make more sense to just redirect them to `INSTALL_HELM.md` in the beginning? If they've gotten to this point, have they already gone down the wrong path?

Reply:
I am ok with either. I think because it was added last I put it at the bottom IIRC. It is a little weird because both this and the link above for the prerequisites just redirect to INSTALL_HELM.md. Let me know what you prefer and I will change it.

Reply:
If you prefer, we can just put a single header at the top that says refer to `INSTALL_HELM.md` to install on Kubernetes. I was trying to match the organization of the other install methods but the way it is now is redundant as you noted.

Reply:
Let me know what you think in the last commit.